paasta_tools/cli/cmds/status.py

```diff
@@ -431,10 +431,16 @@ def paasta_status(args):
                     system_paasta_config=load_system_paasta_config(),
```

Comment:
Could you use this opportunity and move load_system_paasta_config() higher, so that we don't reload it for every cluster?